src/includes/configuration/session-tracking-interval/android.mdx

```diff
@@ -3,3 +3,14 @@
   <meta-data android:name="io.sentry.session-tracking.timeout-interval-millis" android:value="30000" />
 </application>
 ```
+
+Or, if you are manually instrumenting Sentry:
+
+```java
+import io.sentry.android.core.SentryAndroid;
+
+SentryAndroid.init(
+  options -> {
+    options.setSessionTrackingIntervalMillis(10000);
```

Comment:
`l`: I think it would be nice if this matches with the above value in the manifest of `30000`.

Reply:
good catch, i will change the value above to 10k then, as 30k is the default anyway